pkgs/tools/misc/bkyml/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, python3
+}:
+
+with python3.pkgs; buildPythonApplication rec {
+  pname = "bkyml";
+  version = "1.4.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "01kpx35572mp4xl2fjyvfk39jfgfjcyzymbifk76891kaqkjb7r9";
+  };
+
+  # The pyscaffold is not a runtime dependency but just a python project bootstrapping tool. Thus,
+  # instead of implement this package in nix we remove a dependency on it and fix up the version
+  # of the package, that has been affected by the pyscaffold package dependency removal.
+  postPatch = ''
+    substituteInPlace setup.py --replace "['pyscaffold>=3.0a0,<3.1a0'] + " ""
+    substituteInPlace setup.py --replace "use_pyscaffold=True" ""
```

Comment:
```suggestion
    substituteInPlace setup.py \
      --replace "['pyscaffold>=3.0a0,<3.1a0'] + " ""
      --replace "use_pyscaffold=True" ""
```

Reply:
Sorry, the suggestion will break the derivation. A new line character (`\`) needs to be used for all the connected command's lines, otherwise, it will be broken.

But I think I've got the idea and I've done it in f80ee2a, but with valid new line syntax, is that what you meant?